### ISSUE TITLE
Thanks!

### ISSUE THREAD

**Issue by coderschoolreview** [scripts1988/qchat#1]
Hi @scripts1988, please update your work and README file as soon as possible. Thank you .

**Comment by scripts1988** [comment]
Hello, I updated the latest status in Readme and all source code to github already.

**Comment by coderschoolreview** [comment]
Thanks for the code updates @scripts1988, but your README still only have 4 check boxes :) Could you update it?

**Comment by scripts1988** [comment]
Yes, sure. I never stop try my best to complete all mandatory requirement.

**Comment by coderschoolreview** [comment]
hi @script1988, the heroku link still missing from README file. Please update it so we can review your app.

**Comment by scripts1988** [comment]
Hi, I updated heroku url.
